arquillian-ape-sql/standalone/dbunit/src/main/java/org/arquillian/ape/rdbms/dbunit/DbUnitOptions.java

```diff
@@ -0,0 +1,100 @@
+package org.arquillian.ape.rdbms.dbunit;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Set;
+
+public class DbUnitOptions implements Map<String, Object> {
```

Comment:
I am still not sure if implementing map is really the right thing to do. Long term we should think about some other way of exposing configuration.